### PR TITLE
[MIG][12.0] base_delivery_carrier_files

### PR DESCRIPTION
**Pull request by MPoch-PlanetaTIC**
Migration of base_delivery_carrier_files to Odoo 12.0

cc @PlanetaTIC

### CODE REVIEW

File: base_delivery_carrier_files/wizards/generate_carrier_files.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Author: Guewen Baconnier
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning
+
+
+class DeliveryCarrierFileGenerate(models.TransientModel):
+
+    _name = 'delivery.carrier.file.generate'
```
Comment (rousseldenis) [inline base_delivery_carrier_files/wizards/generate_carrier_files.py:12]: Put a _description (see runbot errors)

File: base_delivery_carrier_files/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012 Camptocamp SA
+# Author: Guewen Baconnier
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
```
Comment (rousseldenis) [inline base_delivery_carrier_files/models/stock_move.py]: Change all openerp by odoo

File: base_delivery_carrier_files/models/delivery_carrier_file.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import os
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning
+from ..generator import new_file_generator
+
+
+class DeliveryCarrierFile(models.Model):
+    _name = 'delivery.carrier.file'
```
Comment (rousseldenis) [inline base_delivery_carrier_files/models/delivery_carrier_file.py:15]: Please put a _description

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

File: base_delivery_carrier_files/wizards/generate_carrier_files.py (hunk 1 above)
Comment (MPoch-PlanetaTIC) [reply base_delivery_carrier_files/wizards/generate_carrier_files.py:12]: Done.
I had errors on reading runbot errors. But at least I could read them and solve all the problems.
Thank you for your attention.

File: base_delivery_carrier_files/models/stock_move.py (hunk 2 above)
Comment (MPoch-PlanetaTIC) [reply base_delivery_carrier_files/models/stock_move.py]: Done

File: base_delivery_carrier_files/models/delivery_carrier_file.py (hunk 3 above)
Comment (MPoch-PlanetaTIC) [reply base_delivery_carrier_files/models/delivery_carrier_file.py:15]: Done

File: base_delivery_carrier_files/wizards/generate_carrier_files.py [hunk 4]
```diff
@@ -3,8 +3,8 @@
 # Author: Guewen Baconnier
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields, exceptions, api
-from odoo.tools.translate import _
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning
```
Comment (rousseldenis) [inline base_delivery_carrier_files/wizards/generate_carrier_files.py]: Warning is deprecated. Use UserError instead

File: base_delivery_carrier_files/models/delivery_carrier_file.py [hunk 5]
```diff
@@ -6,8 +6,8 @@
 import os
 import logging
 
-from odoo import api, exceptions, fields, models
-from odoo.tools.translate import _
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning
```
Comment (rousseldenis) [inline base_delivery_carrier_files/models/delivery_carrier_file.py]: Warning is deprecated. Use UserError instead

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

**Comment by rousseldenis** [comment]
@MPoch-PlanetaTIC Could you check flake8 error?

**Comment by MPoch-PlanetaTIC** [comment]
> @MPoch-PlanetaTIC Could you check flake8 error?

Done.
Sorry for my flake8 error, my mistake. Thanks for your feedback

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Comment by ps-tubtim** [comment]
@MPoch-PlanetaTIC Please squash commits before merge.

**Review by thomaspaulb — APPROVED** [review]
Code review. Thanks for this migration!

If you have time, maybe you could review the (still open) 10.0 branch in order to get it merged also. We've been using it in production since 2 years.

**Review by ps-tubtim — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Oh dear, such a delay on a good PR. That's a shame. Seems like I can't restart the Travis build. I'm going to try ocabot, but if it does not work I'm going to ask you to rebase.

/ocabot merge nobump

**Comment by StefanRijnhart** [comment]
It worked! So much :green_heart: for the bot. Thanks for the port @MPoch-PlanetaTIC!

**Comment by MPoch-PlanetaTIC** [comment]
> It worked! So much green_heart for the bot. Thanks for the port @MPoch-PlanetaTIC!

You are welcome ;)

**Pull request merged**